pkg/client/group_mgmt_client.go

```diff
@@ -15,22 +15,26 @@ import (
 )
 
 const (
-	groupURIFmt     = "https://%s:5392/%s"
-	clientTimeout   = time.Second * 60 // 1 Minute
-	maxLoginRetries = 1
-	jobTimeout      = time.Second * 300 // 5 Minute
-	jobPollInterval = 5 * time.Second   // Second
-	smAsyncJobId    = "SM_async_job_id"
+	groupURIFmtTenant = "https://%s:443/api/tenant/%s"
+	groupURIFmt       = "https://%s:5392/%s"
+	clientTimeout     = time.Second * 60 // 1 Minute
```

Comment:
We could use time.Minute and remove the comment.  Similar for the other time constants here.